src/carcockpit/shaders/shader_pbr.hpp

```diff
@@ -21,28 +21,33 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 #pragma once
 
-// #include <ruis/render/texturing_shader.hpp>
+#include <ruis/config.hpp>
 #include <ruis/render/opengles/shader_base.hpp>
 #include <ruis/render/texture_2d.hpp>
+#include <ruis/render/texture_cube.hpp>
 
-namespace ruis {
-using mat3 = r4::matrix3<real>;
-using matrix3 = mat3;
-static_assert(sizeof(mat3) == sizeof(real) * 3 * 3, "size mismatch");
-} // namespace ruis
+namespace ruis::render {
 
-namespace carcockpit {
-
-class shader_adv : public ruis::render::opengles::shader_base
+/**
+ * @brief Physically based rendering (PBR) is a computer graphics approach that seeks to render images in a way that
```

Comment:
The `@brief` in doxygen should describe the class/function/... briefly in one line. The next line after is where detailed description starts.
So, this comment should be formatted like this:
```
/**
 * @brief Physically based rendering (PBR) shader.
 * Physically based rendering (PBR) is a computer graphics approach that seeks
 * to render images in a way that models the lights and surfaces with optics in the real world.
 * Current PBR shader implementation uses the following texture information:
 * - Diffuse (aka albedo)
 * - Normals (per-fragment micro-relief)
 * - Ambient, for pre-baked ambient occlusion
 * - Roughness Metalness
 * For more info, see: https://en.wikipedia.org/wiki/Physically_based_rendering
 */
```